src/declarations/stencil-public-compiler.ts

```diff
@@ -302,9 +302,18 @@ export interface ConfigExtras {
    * loading components when using a bundler such as Vite or Parcel. Setting this flag to `true` will change how Stencil
    * lazily loads components in a way that works with additional bundlers. Setting this flag to `true` will increase
    * the size of the compiled output. Defaults to `false`.
+   * @deprecated Use `enableImportInjection`
    */
   experimentalImportInjection?: boolean;
 
+  /**
```

Comment:
I copied this 1:1 without the experimental part. Any idea how to improve?

Reply:
I think this is good 👍